### PR TITLE
Remove support for EGD

### PR DESCRIPTION
**Pull request by richsalz**
## Checklist
Thank you for your contribution. Please answer the following:

- [X] I acknowledge that I am authorized to submit this code under
the terms of the [Apache License](https://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Review by botovq — CHANGES_REQUESTED** [review]
Please define `OPENSSL_NO_EGD` in `configuration.h.in`, and remove the `EGD_socket` definition in `doc/man1/tsget.pod` other than that this is ok.

**Comment by richsalz** [comment]
Ah, right the compatibility OPENSSL_NO_xxx flags.  Added, commit pushed.

File: include/openssl/configuration.h.in [hunk 1]
```diff
@@ -24,6 +24,7 @@ extern "C" {
 /*
  * OpenSSL was configured with the following options:
  */
+#define OPENSSL_NO_EGD
```
Comment (botovq) [inline include/openssl/configuration.h.in:27]: I'd move this down to the end next to the `NO_EC2M` one that Watson added.

Comment (richsalz) [reply include/openssl/configuration.h.in]: Done.  (And I tidied up the EC2M spacing while I was there.)

**Review by botovq — APPROVED** [review]

**Pull request merged**